pkg/framework/plugins/nodeutilization/nodeutilization.go

```diff
@@ -752,3 +753,19 @@ func assessAvailableResourceInNodes(
 
 	return available, nil
 }
+
+// WithResourceRequestForAny returns a filter function that checks if a pod
+// has a resource request specified for any of the given resources names.
+func WithResourceRequestForAny(names ...v1.ResourceName) pod.FilterFunc {
```

Comment:
This can stay private as it is used only within the package.

Reply:
done.